File: src/test/testData/java/implicit/Queue.java
package ru.ustinov;

import java.net.URL;
import java.util.HashSet;
import java.util.Queue;

class Simple {
    public static void main() {
        <warning descr="This data structure can probably implicitly invoke URL#hashCode()">Queue<URL> a;</warning>
        Queue<Integer> b;
        <warning descr="This data structure can probably implicitly invoke URL#hashCode()">Queue<HashSet<URL>> c, d;</warning>
        Queue<HashSet<Integer>> e, f;
    }
}

File: src/test/testData/kotlin/implicit/Queue.kt
import java.net.URL
import java.util.HashSet
import java.util.Queue

@Suppress("UNUSED_VARIABLE")
fun main() {
    <warning descr="This data structure can probably implicitly invoke URL#hashCode()">val a: Queue<URL></warning>
    val b: Queue<Int>
    <warning descr="This data structure can probably implicitly invoke URL#hashCode()">val c: Queue<HashSet<URL>></warning>
    val d: Queue<HashSet<Int>>
}